spot_rl_experiments/experiments/skill_test/run_episode.py

```diff
@@ -0,0 +1,131 @@
+# mypy: ignore-errors
```

Comment:
what is the script? is this running nav-pick-nav-place?

Reply:
yeah Tushar helped create this script for us to run nav-pick-nav-place for the skills benchmarking. It also has functionalities to catch termination signals and save the logs from the experiment so far.